NEWS

```diff
@@ -65,6 +65,28 @@ New Functionality
 Changed Functionality
 ---------------------
 
+- Logging of failed analyzers has been overhauled. `dpd.log` was replaced with
+  a new `analyzer_failed.log` that presents a more unified and consistent view
+  of failed analyzers.
+
+  For protocol analyzers, `analyzers_failed.log` contains analyzers that
+  reported a protocol violation that caused Zeek to decide to remove the violating
+  analyzer from the connection.
```

Comment:
I saw it's in the PR cover letter - probably should mention the "analyzer was previously confirmed" here as well. I guess everyone except for me is expecting/assuming that anyhow. Might still be good to make explicit mention here and/or somewhere in the docs that failure and removal of non-confirmed protocol analyzers aren't part of the log because these aren't interesting (dpd.log always worked like that, but analyzer.log didn't).